docker/generic/start_proxy.py

```diff
@@ -887,6 +887,17 @@ def make_argparser():
         envoy. Only change if running multiple ESPv2 instances on the same host.
         '''
     )
+    parser.add_argument(
+        '--config_yaml',
```

Comment:
maybe call it --envoy-extra-config-yaml?   

It is the extra boostrap config in additional to ESPv2 clusters/listeners.  The cluster names and listener name should not step over the ones generated by ESPv2, otherwise it will break ESPv2. 

I am really hesitate to add this flag, if it is not used carefully, we could shot yourself in the foot.   Could you provide some config examples that you like to use this flag for?

Reply:
Hi, 
The main goal is to enable all Envoy's abilities - Gzip configuration in my case - independently (and, in addition, to allow dynamic management to every aspect of the bootstrap config).

We can also name it with a more "alarming" name such as `--modify-bootstrap-conig-yaml` and highlight the "overriding" and "merging" aspects of it in the help text - although I am pretty sure that users will mostly use it to manage static resources that managing them isn't being supported yet through ESP (like in my case)

Reply:
I prefer to call it `--envoy-extra-config-yaml` and ok to add this flag.